Gosu/Audio.hpp

```diff
@@ -80,59 +90,7 @@ namespace Gosu
         //! or low values. Use 1.0 for normal playback speed.
         Channel play_pan(double pan, double volume = 1, double speed = 1,
             bool looping = false) const;
-    };
-
-    //! Songs are less flexible than samples. Only Song can be played at any given time,
-    //! and there is no way to control its pan (stereo position) or speed.
-    class Song
```

Comment:
Should Song be available in C++ for backwardscompat as well?

Reply:
Hmmm...I would say yes. At least for now :)